airbyte-integrations/connectors/source-snapchat-marketing/unit_tests/unit_test.py

```diff
@@ -0,0 +1,77 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+import json
+from typing import Any, Mapping
+
+from airbyte_cdk.models import SyncMode
+from source_snapchat_marketing.source import Adaccounts, Organizations, SnapchatAdsOauth2Authenticator, get_depend_on_ids
+
+
+def _config() -> Mapping[str, Any]:
+    """
+    Get the config from /test_input
+    """
+    with open("secrets/config.json", "r") as f:
+        return json.loads(f.read())
+
+
+config = _config()
+created_streams = {}
+
+kwargs = {"sync_mode": SyncMode.incremental, "cursor_field": "updated_at", "stream_state": None}
+
+depends_on_stream_config = {"authenticator": SnapchatAdsOauth2Authenticator(config), "start_date": "1970-01-01"}
+
+expected_organization_ids = [{"organization_id": "7f064d90-52a1-42db-b25b-7539e663e926"}]
+expected_adaccount_ids = [
+    {"ad_account_id": "04214c00-3aa5-4123-b5c8-363c32c40e42"},
+    {"ad_account_id": "e4cd371b-8de8-4011-a8d2-860fe77c09e1"},
+]
+
+
+def test_get_depend_on_ids_none():
+    """ Testing the stream that has 1 level parent dependency (like Adaccounts has dependency) """
```

Comment:
think this docstring is for test_get_depend_on_ids_1(). Can you update for this method?

Reply:
Yes, my bad. Updated for `...ids_none` method. Actually I updated all the docstrings for unit_tests